src/Board/Board.h

```diff
@@ -1,12 +1,16 @@
 #pragma once
 
-#include <stdint.h>
 #include <Pieces/Piece.h>
+#include <stdint.h>
+
+#define ROW_BOUNDS(bitshift) (bitshift >= 0 && bitshift <= 28)
+#define COLUMN_BOUNDS(arr_idx) (arr_idx >= 0 && arr_idx <= 7)

```

Comment:
I think rows and columns are swapped by mistake here

Reply:
🤓